src/server/middleware/sendHtml.js

```diff
@@ -40,8 +40,8 @@ function getChunkAssets(assetsByChunkName) {
     .map(([, assets]) => (typeof assets === 'string' ? assets : assets[0]));
 }
 
-const modernBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.webpack-stats.browser.json').assetsByChunkName);
-const legacyBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.webpack-stats.legacyBrowser.json').assetsByChunkName)
+const modernBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.build-meta.json').modernBrowserChunkAssets);
```

Comment:
Made changes in `one-app-bundler` to add all the necessary data previously taken from `webpack-stats-*` files to the `build-meta.json` file. Doing so means we no longer have to add `webpack-stats-*` files to the docker image and prevents memory issues from loading these very large files into memory.

As a result of this change our docker image size went down from ~397mb to ~190mb`.